src/winetricks

```diff
@@ -738,6 +738,26 @@ w_get_github_latest_release()
     echo "$latest_version"
 }
 
+# Get the latest tagged prerelease from github.com API
+w_get_github_latest_prerelease()
+{
+    # $1 - github organization
+    # $2 - github repository
```

Comment:
To be a bit clearer, please do:
```
org="$1"
repo="$2"
...
WINETRICKS_SUPER_QUIET=1 w_download_to "${W_TMP_EARLY}" "https://api.github.com/repos/${org}/${repo}/releases" "" "release.json" >/dev/null 2>&1
```

Reply:
Done

Reply:
Thanks (and for fixing w_get_github_latest_release() ).